tslint.json

```diff
@@ -49,7 +49,8 @@
       "adjacent-overload-signatures": [false],
       "no-string-literal": [false],
       "max-classes-per-file": [false],
-      "one-variable-per-declaration": [false]
+      "one-variable-per-declaration": [false],
+      "arrow-parens": [false]
```

Comment:
I typically dislike a PR that attempts to sneak in a change like this, but since this one is so awesome, I'll look the other way :-P